### PR TITLE
Reorganized static pred and priorities into Default sections 3.6

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Comment by mburke5678** [comment]
3.9 https://github.com/openshift/openshift-docs/pull/8210
3.7 https://github.com/openshift/openshift-docs/pull/8087
3.6 https://github.com/openshift/openshift-docs/pull/8211

File: admin_guide/scheduling/scheduler.adoc [hunk 1]
```diff
@@ -269,54 +269,57 @@ These predicates do not take any configuration parameters or inputs from the
...
 {"name" : "CheckNodeMemoryPressure"}
 ----
 
 **_CheckNodeDiskPressure_** checks if a pod can be scheduled on a node with a disk pressure condition.
-_The default scheduler policy includes this predicate._
 ----
 {"name" : "CheckNodeDiskPressure"}
 ----
 
+[other-predicates]]
```
Comment (vikram-redhat) [inline admin_guide/scheduling/scheduler.adoc]: @mburke5678 - this doesn't look correct. Can you check in other branches as well?

Comment (mburke5678) [reply admin_guide/scheduling/scheduler.adoc]: @vikram-redhat Will do.

Comment (mburke5678) [reply admin_guide/scheduling/scheduler.adoc]: Fixed https://github.com/openshift/openshift-docs/pull/8219

**Pull request merged**